scripts/fix-aria-hidden.sh

```
@@ -0,0 +1,46 @@
+count=$(grep -r 'aria-hidden="true".*aria-hidden="true"' frontend/src --include="*.tsx" --include="*.ts" | wc -l)
+
+if [ "$count" -gt 0 ]; then
+  echo "⚠️  Found $count files with aria-hidden duplicates"
+  echo ""
+  echo "Files with issues:"
+  grep -r 'aria-hidden="true".*aria-hidden="true"' frontend/src --include="*.tsx" --include="*.ts" -l
+  echo ""
+  echo "🔧 Fixing duplicates..."
+  
+  # Fix pattern: aria-hidden="true" ... aria-hidden="true"
+  find frontend/src -name "*.tsx" -o -name "*.ts" | xargs sed -i 's/aria-hidden="true" weight="bold" aria-hidden="true"/weight="bold" aria-hidden="true"/g'
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)